public/index.html

```diff
@@ -25,6 +25,24 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <!-- Crystal-ui components library link below
+    🔥  if you want to look into it check 🔛  : https://crystal-ui.netlify.app/
+    -->
+    <link
+      rel="stylesheet"
+      href="https://crystal-ui.netlify.app/components.css"
+    />
+    <!-- 🔗 Mockman js styles link below -->
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/mockman-js@latest/dist/style.css"
+    />
+    <!-- 🔥 Below is a link for google icons for all cases outlined and filled, sharp
+    all 🔥 -->
+    <link
+      href="https://fonts.googleapis.com/css?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Two+Tone|Material+Icons+Round|Material+Icons+Sharp"
+      rel="stylesheet"
+    />
```

Comment:
Oh, so this way we can use material icons of all styles. This is great! Thank you!